challenges.md

```diff
@@ -222,3 +222,32 @@ Candidates interested in becoming our first digital content strategist, we would
     - Tags for searchability and indexing
     - Please write the sample in [Markdown](https://en.wikipedia.org/wiki/Markdown)
 
+# Integrations Developer
+
+We have three projects to choose from if you're applying to be an integrations developer at Contrast.
+
+Pick a project that appeals to you and follow the instructions. Commit often with quality commit messages and so that we can see your thought process as the project progresses.
```

Comment:
I don't think we need 'and' in the second sentence.

Reply:
👍